themes/default/content/resources/getting-started-with-1password-and-pulumi/index.md

```diff
@@ -58,26 +58,22 @@ main:
     datetime: ""
     # Description of the webinar.
     description: |
-      If you’re using 1Password to manage keys, account passwords and other sensitive information you can now access those secrets from the Pulumi CLI using the new Pulumi 1Password plugin.
+      Software developers need to secure their workflows to prevent the accidental exposure of API keys and other secrets. 1Password is known as a capable password manager, but 1Password Developer Tools also provide a set of capabilities to secure secrets throughout the software development lifecycle. 
```

Comment:
```suggestion
      Software developers need to secure their workflows to prevent the accidental exposure of API keys and other secrets. 1Password is known as a capable password manager, but [1Password Developer Tools](https://developer.1password.com/) also provide a set of capabilities to secure secrets throughout the software development lifecycle. 
```